app/Http/Controllers/API/AuthAPIController.php

```diff
@@ -107,7 +125,7 @@ public function login(Request $request)
         }
 
         // no errors, return the token
-        return Response::json(['token' => $token->get()]);
+        return Response::json(['token' => $token->get() , 'user_id' => $volunteer->id]);
```

Comment:
As an unique identifier to the current user when traversing through any users array.
for instance , when a user opens an organization profile I use the id to look if the user is already a subscriber or not , using that I can show him proper subscribe/unsubscribe buttons.